src/patternfly/components/Badge/badge.scss

```diff
@@ -63,10 +63,13 @@
     --#{$badge}__toggle-icon--Color: var(--#{$badge}--m-unread__toggle-icon--Color);
   }
 
+  &:disabled,
```

Comment:
This is fine but AFAIK the badge is never an interactive element, so `:disabled` is likely never applied? Same for `pointer-events`, the badge should (currently anyways) go in something that's disabled and not be disabled itself, so pointer-events shouldn't be needed, but shouldn't cause a problem, either.

Reply:
✌️ unneeded code bloat

Reply:
I disagree, I can easily see a badge itself being interactive. I'll provide an example in a bit.